javascript/ql/src/Security/CWE-754/UnvalidatedDynamicMethodCall.qhelp

```diff
@@ -11,8 +11,8 @@ could choose a name that makes the application invoke an unexpected method, whic
 exception. If this exception is not handled, it could be used to mount a denial-of-service attack.
 </p>
 <p>
-For example, there might not be a method of the given name or the result of the lookup might not be
```

Comment:
Line 9: "if he method name is user controlled" -> if **t**he method name is user**-**controlled